worker/jobTypes/S3Publish.js

```diff
@@ -69,11 +72,39 @@ class S3PublishClass {
       const exec = workerUtils.getExecPromise();
       const command = deployCommands.join(' && ');
       const { stdout } = await exec(command);
-      let stdoutMod = '';
-      // get only last part of message which includes # of files changes + s3 link
-      if (stdout.indexOf('Summary') !== -1) {
-        stdoutMod = stdout.substr(stdout.indexOf('Summary'));
+      let stdoutMod = stdout;
+
+      // check if json was returned from mut
+      try {
+        const stdoutJSON = JSON.parse(stdout);
+        const urls = stdoutJSON.urls;
+        // pass in urls to fastly function to purge cache
+        this.fastly.purgeCache(urls).then(function(data) {
```

Comment:
This function is async -- could we await here instead of promise-chaining?

Reply:
Explained offline -- we do not want to actually stop execution of this function to wait for the purge to finish